hack/config.sh

```diff
@@ -2,8 +2,8 @@ unset binaries docker_images docker_tag docker_tag_alt image_prefix image_prefix
     namespace image_pull_policy verbosity \
     csv_version package_name bazel_py
 
-source ${KUBEVIRT_PATH}hack/config-default.sh
-source ${KUBEVIRT_PATH}cluster-up/hack/config.sh
+source hack/config-default.sh
```

Comment:
Why is this change required?

Reply:
these paths were wrong when I tested, this cleared up the issue, but I'll test again, make sure it's still needed.